packages/@aws-cdk/aws-ecs/test/test.container-definition.ts

```diff
@@ -1293,5 +1293,88 @@ export = {
       test.done();
     }
   },
-  // render extra hosts test
+
+  "Firelens Configuration": {
+    "simple fluent-bit log router container"(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+      const taskDefinition = new ecs.Ec2TaskDefinition(stack, 'TaskDef');
+
+      new ecs.ContainerDefinition(stack, "log_router", {
```

Comment:
See above about having this wrapped up as a method on TaskDefinition

Reply:
TaskDefinition.addFirelensLogRouter has been added: https://github.com/aws/aws-cdk/pull/5457/files#diff-5224292ec401f169f9612a0520276ff1R372